allennlp/service/server_sanic.py

```diff
@@ -6,8 +6,8 @@
 
 # Move access.log and error.log to /tmp by default
 # If someone really wants them, they can move them back
-LOGGING['handlers']['accessTimedRotatingFile']['filename'] = '/tmp/sanic_access.log'
-LOGGING['handlers']['errorTimedRotatingFile']['filename'] = '/tmp/sanic_error.log'
+# LOGGING['handlers']['accessTimedRotatingFile']['filename'] = '/tmp/sanic_access.log'
+# LOGGING['handlers']['errorTimedRotatingFile']['filename'] = '/tmp/sanic_error.log'
```

Comment:
I will remove those lines.  @matt-gardner you had brought up this cause in the past (upgrading sanic) and I found it to be true.  I don't know how, but I repeatedly bust my conda environment.